src/cli/util.js

```diff
@@ -403,6 +403,7 @@ function eachFilename(context, patterns, callback) {
   if (ignoreNodeModules) {
     patterns = patterns.concat(["!**/node_modules/**", "!./node_modules/**"]);
   }
+  patterns = patterns.concat(["!**/.{git,svn,hg}/**", "!./.{git,svn,hg}/**"]);
```

Comment:
It seems the `!./.{git,svn,hg}/**` part is unnecessary, and so does `!./node_modules/**`.

```sh
$ tree test
test
├── dir
│   ├── node_modules
│   │   └── x.js
│   └── x.js
├── node_modules
│   └── x.js
└── x.js

$ node -e 'console.log(
  require("globby").sync(
    ["**/*.js", "!**/node_modules/**"],
    { cwd: "test" }
  )
)'
[ 'dir/x.js', 'x.js' ]
```

Reply:
#2185

Reply:
Try re-running that with `./**/*.js` as the glob.

Reply:
Cool, didn't know that. Can you add a comment for it?

Reply:
Sure.